### PR TITLE
Add 3.9.4 release notes

### PR DESCRIPTION
**Pull request by crolopez**
This PR adds the release notes for v3.9.4.

It still remains the section for the Wazuh API and Wazuh APP.

Regards.

### CODE REVIEW

File: source/release-notes/release_3_9_4.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- `wazuh/wazuh <https://github.com/wazuh/wazuh/blob/v3.9.4/CHANGELOG.md>`_
+- `wazuh/wazuh-kibana-app <https://github.com/wazuh/wazuh-kibana-app/blob/v3.9.4-7.2.0/CHANGELOG.md>`_
+- `wazuh/wazuh-splunk <https://github.com/wazuh/wazuh-splunk/blob/v3.9.4-7.3.0/CHANGELOG.md>`_
+
+
+Wazuh core
+----------
+
+- The attributes of FIM alerts will no longer depend on race conditions between realtime and whodata modes.
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: Avoid a race condition was an issue, but the problem that the user saw was that FIM could apply the parent file's option set for a monitored file.

File: source/release-notes/release_3_9_4.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Wazuh-DB will delete all information stored about an agent immediately after its removal.
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: A user expects this to be that way. What was happening before?

File: source/release-notes/release_3_9_4.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Avoided critical errors in Remoted when handling invalid connections
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: The problem here was that Remoted may handle a message from an agent that has just disconnected.

Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: Missing period.

File: source/release-notes/release_3_9_4.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+- Fixed a bug when processing maximum length IPs in agents.
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: What is the impact of this issue? Database, Kibana App...

File: source/release-notes/release_3_9_4.rst [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- Logcollector will prioritize the file monitoring options specified in the shared configuration.
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: The user may understand this incorrectly. Does "prioritize" mean that Logcollector reads the local-files defined in the shared configuration before the ones declared in the local configuration?

The thing is that Logcollector should let local-file stanzas in the shared configuration overwrite older entries if they have the same `<location>` value.

File: source/release-notes/release_3_9_4.rst [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+- Analysisd will not break due to a race condition when handling JSON objects.
+- SCA wodle will not generate inconsistencies in the database when policy IDs are duplicated.
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: "wodle" is old. Please leave SCA only, and clarify that this fix affects the manager only.

File: source/release-notes/release_3_9_4.rst [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+- The cluster engine will not have a race condition with Remoted when synchronizing some files.
+- Avoided the possibility of generating a handler leak when using whodata mode for Windows.
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: I think this sentence is too complex. I would say something like "A handler leaking hazard has been fixed in the FIM who-data engine on Windows".

File: source/release-notes/release_3_9_4.rst [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+Wazuh apps
+----------
+
```
Comment (vikman90) [inline source/release-notes/release_3_9_4.rst]: Something missing here?

Comment (bah07) [reply source/release-notes/release_3_9_4.rst]: @juankaromo has updated this section.

**Pull request merged**